services/app-web/src/contexts/TraceContext.tsx

```diff
@@ -49,9 +48,6 @@ registerInstrumentations({
     ],
 })
 
-BaseOpenTelemetryComponent.setTracer(serviceName)
```

Comment:
Thanks for stripping this out I was thinking we didn't need it but didnt want to touch the TraceContext